app/viewmodels/checkAnswers/sections/guarantor/GuarantorArrangerSummary.scala

```diff
@@ -44,7 +44,7 @@ object GuarantorArrangerSummary {
             controllers.sections.guarantor.routes.GuarantorArrangerController.onPageLoad(
               ern = request.ern,
               draftId = request.draftId,
-              mode = CheckMode
+              mode = NormalMode
```

Comment:
I think we still want this to be `CheckMode` and we'd only change it to `NormalMode` in `GuarantorArrangerController.onSubmit` if they change their answer